ui/components/app/edit-gas-fee-popover/network-statistics/index.scss

```diff
@@ -7,37 +7,32 @@
     height: 56px;
     display: flex;
     align-items: center;
-    justify-content: space-around;
```

Comment:
I cleaned up this CSS slightly. Some classes looked like `.foo__bar__baz`, which is illegal in BEM, and were cleaned up to just `.foo__baz`. I also got rid of the separator divs, choosing to just use borders instead, as well as the `width: 30%` that was getting applied to each div, choosing to just let flexbox do its magic. The result is that certain sections of the Network Status area can disappear and the UI will still look good because it will adapt automatically.